umake/frameworks/ide.py

```diff
@@ -857,6 +857,41 @@ def post_install(self):
                         categories="Development;TextEditor;"))
 
 
+class DBeaver(umake.frameworks.baseinstaller.BaseInstaller):
+
+    def __init__(self, **kwargs):
+        super().__init__(name="DBeaver", description=_("Free multi-platform database tool for developers"),
+                         only_on_archs=['i386', 'amd64'],
+                         download_page="https://dbeaver.io/download/",
+                         desktop_filename="dbeaver.desktop",
+                         required_files_path=["dbeaver"],
+                         dir_to_decompress_in_tarball="dbeaver_*",
+                         **kwargs)
+
+    arch_trans = {
+        "amd64": "x64",
```

Comment:
the architectures are x86_64 and x86 respectively (the ones you can see in the webpage in the previous comment